test/resource/http.js

```diff
@@ -93,7 +92,7 @@ exports.server = function _listen(ready) {
 
     req.on('data', (chunk) => chunks.push(chunk));
     req.on('end', (chunk) => {
-      if (chunk) chunks.push(chunk);
+      if (chunk) { chunks.push(chunk); }
```

Comment:
I don't know why this is causing your coverage to go down, but if you have a chance, you should get the coverage back up.